docs/setup/native.md

```diff
@@ -90,6 +90,12 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `brew install redis`
 1. Install Postgresql & PostGIS
    1. It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) which installs the correct combination of Postgresql and PostGIS versions.
+   	  - Navigate to: https://postgresapp.com/downloads.html
```

Comment:
This line seems to duplicate the link above

Reply:
Updated ✔️